SocialFilter/sections/general_elemhide.txt

```diff
@@ -15,17 +15,17 @@
 !+ NOT_OPTIMIZED
 ~facebook.com##a[href^="fb-messenger://share"]
 !+ NOT_OPTIMIZED
-~twitter.com##a[href^="https://twitter.com/intent/tweet?"]
```

Comment:
I guess that it's still used.
For example, it's required here - `https://www.urbandictionary.com/`
I suppose that it might take some time until it will be replaced on all websites, so I think that we should keep old rules.

Reply:
Do you want to keep old rules and add new rules with `x.com`?

Reply:
Yes, I think that we should keep old rules and we can additionally add new rules.

Reply:
Also agree with that.

Reply:
Discussed in https://listauthorschat.slack.com/archives/C010N14G4TF/p1715941462620859?thread_ts=1715923566.904009&cid=C010N14G4TF IMO we should add new `x.com` rules one by one on actual needs; however, `~x.com` should be added wherever `~twitter.com` is in place.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/180173/files#r1615783654
They are different.
A rule existing in line 18 is about `x.com` and in line 48 is about `twitter.com`.

Reply:
Btw, what about grouping the rules under a comment? Like this
```
! X (formely Twitter)
rules
```

Reply:
I oppose adding `x.com` counterparts to all exisiting `twitter.com` rules. These are heavier side of generic cosmetic rules.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/180173#discussion_r1615793084
https://github.com/AdguardTeam/AdguardFilters/pull/180173/commits/cca19a04ab505adc7f8406a28d8bc3a03d59928b

Reply:
@Yuki2718 Did you see `/intent/tweet?` on X? I don't see, sharing works without this link.

Reply:
Not sure what you mean, don't you see in the linked issue https://github.com/AdguardTeam/AdguardFilters/issues/180172 ?